app/models/registration.rb

```diff
@@ -28,6 +28,10 @@ class Registration < ApplicationRecord
   validate :registration_limit_not_exceed, on: :create
   validate :registration_to_events_only_if_present
 
+  validates :accepted_code_of_conduct, acceptance: {
+    unless:  -> { conference.code_of_conduct.blank? }
```

Comment:
`if` + `present?` - it is more readable and consistent with style guides and Rubocop configuration :wink: